### PR TITLE
(refactor) refactor mealpage and meallist with material ui styling

### PR DESCRIPTION
**Pull request by rs214**

### CODE REVIEW

File: client/src/containers/mealDetails.js [hunk 1]
```diff
@@ -2,6 +2,8 @@ import React, { Component } from 'react';
 import { connect } from 'react-redux';
 import { fetchMealDetail } from '../actions/index';
 import { Link } from 'react-router-dom';
+import {Card, CardActions, CardHeader, CardMedia, CardTitle, CardText} from 'material-ui/Card';
```
Comment (gzeegz) [inline client/src/containers/mealDetails.js:5]: Add spaces to the beginning and end when you destructure here, i.e., `{ Card, ..., CardText }`

File: client/src/containers/mealDetails.js [hunk 2]
```diff
@@ -18,13 +20,30 @@ class MealDetails extends Component {
...
+        <CardText>
+          {this.props.meal.description}
+        </CardText>
+        <CardTitle subtitle="Pickup Information"/>
+        <CardText>
+          {this.props.meal.pickupInfo}
+        </CardText>
+        <CardActions>
+          <FlatButton label="Purchase" />
+          <FlatButton label={<Link to={"/meals"} style={{color: "black", textDecoration: "none"}}>Back To Meals</Link>} />
```
Comment (gzeegz) [inline client/src/containers/mealDetails.js:44]: The quotes are in a JavaScript block, so should be replaced with single quotes.

Also, is there a way to rewrite this so we're not passing a component as a label?

File: client/src/containers/mealDetails.js [hunk 3]
```diff
@@ -18,13 +20,30 @@ class MealDetails extends Component {
...
+        </CardActions>
+      </Card>
       </div>
```
Comment (gzeegz) [inline client/src/containers/mealDetails.js:47]: The `<div>` isn't required if there is only a single child.

File: client/src/containers/mealList.js [hunk 4]
```diff
@@ -3,37 +3,48 @@ import { connect } from 'react-redux';
 import { fetchMeals } from '../actions/index';
 import { Link } from 'react-router-dom';
 import _ from 'lodash';
+import {GridList, GridTile} from 'material-ui/GridList';
```
Comment (gzeegz) [inline client/src/containers/mealList.js:6]: Add some space to the destructuring.

File: client/src/containers/mealList.js [hunk 5]
```diff
@@ -3,37 +3,48 @@ import { connect } from 'react-redux';
...
+          cellHeight={180}
+          style={styles.gridList}
+        >
+          <Subheader>List of Meals</Subheader>
+          {_.map(this.props.meals, (meal) => (
+            <GridTile
+              key={meal.name}
+              title={<Link to={`/meals/${meal.id}`} style={{color:"white", textDecoration: "none"}}>{meal.name}</Link>}
+              subtitle={<span>by <b>JORV</b></span>}
+              actionIcon={<IconButton><StarBorder color="white" /></IconButton>}
```
Comment (gzeegz) [inline client/src/containers/mealList.js:42]: Is there a better way to pass these components as props into `GridTile`?

**Review by gzeegz — CHANGES_REQUESTED** [review]

File: client/src/containers/mealDetails.js (hunk 2 above)
Comment (rs214) [reply client/src/containers/mealDetails.js:44]: The code was from material UI, the label renders the name to that button i think

File: client/src/containers/mealDetails.js (hunk 3 above)
Comment (rs214) [reply client/src/containers/mealDetails.js:47]: fixed

File: client/src/containers/mealDetails.js (hunk 1 above)
Comment (rs214) [reply client/src/containers/mealDetails.js:5]: fixed

File: client/src/containers/mealList.js (hunk 4 above)
Comment (rs214) [reply client/src/containers/mealList.js:6]: fixed

File: client/src/containers/mealList.js (hunk 5 above)
Comment (rs214) [reply client/src/containers/mealList.js:42]: part of material UI, i can refactor the entire thing as a method and call it in the render function  for an easier read but i dont think i can change the props

Comment (gzeegz) [reply client/src/containers/mealList.js:42]: That's okay if it is how Material UI is used.

**Pull request closed without merging**